upgrades/steps_24.go

```diff
@@ -27,5 +27,12 @@ func stateStepsFor24() []Step {
 				return context.State().CreateMissingApplicationConfig()
 			},
 		},
+		&upgradeStep{
+			description: "remove votingmachineids",
+			targets:     []Target{DatabaseMaster},
+			run: func(context Context) error {
+				return context.State().CreateMissingApplicationConfig()
```

Comment:
I missed this copy/paste error when I approved.
I have fixed it in a PR coming shortly for the Mongo space state/name removal.